src/client/app/service/utils/utils.ts

```diff
@@ -0,0 +1,9 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
+export class Utils {
```

Comment:
I _really_ dislike naming things "Utils" -- it reminds me of variables named "data".

Any way to give this class a more meaningful name? Or maybe organize the functions by their usage? Or maybe both?

E.g. call this class "Events"?